ogr/ogrsf_frmts/elastic/ogrelasticdriver.cpp

```diff
@@ -103,6 +103,7 @@ void RegisterOGRElastic() {
     poDriver->SetMetadataItem( GDAL_DMD_CONNECTION_PREFIX, "ES:" );
     poDriver->SetMetadataItem( GDAL_DMD_CREATIONOPTIONLIST,
                                "<CreationOptionList/>");
+    poDriver->SetMetadataItem( GDAL_DMD_SUPPORTED_SQL_DIALECTS, "NATIVE OGRSQL SQLITE" );
```

Comment:
By reading https://gdal.org/drivers/vector/elasticsearch.html I am not sure if "NATIVE" is the default.

Reply:
indeed. changed to "OGRSQL SQLITE ES"